pallets/elections/src/migrations/v1_to_v2.rs

```diff
@@ -0,0 +1,72 @@
+use crate::Config;
+use frame_support::{
+    log, storage_alias,
+    traits::{Get, PalletInfoAccess, StorageVersion},
+    weights::Weight,
+};
+use sp_std::vec::Vec;
+
+// V1 storages
+#[storage_alias]
+pub type MembersPerSession = StorageValue<Elections, u32>;
+#[storage_alias]
+type ReservedMembers<T> = StorageValue<Elections, Vec<<T as frame_system::Config>::AccountId>>;
+#[storage_alias]
+type NonReservedMembers<T> = StorageValue<Elections, Vec<<T as frame_system::Config>::AccountId>>;
+#[storage_alias]
+type ErasMembers<T> = StorageValue<
+    Elections,
+    (
+        Vec<<T as frame_system::Config>::AccountId>,
+        Vec<<T as frame_system::Config>::AccountId>,
+    ),
+>;
+
+// V2 storages
+#[storage_alias]
+pub type CommitteeSize = StorageValue<Elections, u32>;
+#[storage_alias]
+type NextEraReservedValidators<T> =
+    StorageValue<Elections, Vec<<T as frame_system::Config>::AccountId>>;
+#[storage_alias]
+type NextEraNonReservedValidators<T> =
+    StorageValue<Elections, Vec<<T as frame_system::Config>::AccountId>>;
+#[storage_alias]
+type CurrentEraValidators<T> = StorageValue<
+    Elections,
+    (
+        Vec<<T as frame_system::Config>::AccountId>,
+        Vec<<T as frame_system::Config>::AccountId>,
+    ),
+>;
+
+/// This migration refactor storages as follow:
+///
+/// - `MembersPerSession` -> `CommitteeSize`
+/// - `ReservedMembers` -> `NextEraReservedMembers`
+/// - `NonReservedMembers` -> `NextEraNonReservedMembers`
+/// - `ErasMembers` -> `CurrentEraValidators`
+pub fn migrate<T: Config, P: PalletInfoAccess>() -> Weight {
+    log::info!(target: "pallet_elections", "Running migration from STORAGE_VERSION 1 to 2 for pallet elections");
+
+    let writes = 5;
+    let reads = 4;
+
+    let mps = MembersPerSession::get().expect("");
```

Comment:
`expect("")` should be `unwrap()` but in fact there shouldn't be any of them due to no panic policy.

Reply:
fixed, here and in the v0->v1 migration as well.